tests/baselines/reference/api/typescript.d.ts

```diff
@@ -8040,7 +8040,6 @@ declare namespace ts {
     }
     interface SyntaxList extends Node {
         kind: SyntaxKind.SyntaxList;
-        _children: Node[];
```

Comment:
It's not entirely clear to me why this is here, but it wouldn't be too hard code wise to keep it working. I haven't found any evidence that anyone's actually using it...

Reply:
I'm going to guess that it's named this way because it's a special node, but still needs to work with `getChildren`.

Reply:
`SyntaxList` is created during `getChildren` to handle cases where we parsed a `NodeArray` like `modifiers` for a node that doesn't support them (and thus isn't visited by `forEachChild`). I'm not sure why it's in the public API, so it's probably an oversight.